packages/near-snap/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@near-wallet-selector/near-snap",
+  "version": "7.0.2",
```

Comment:
1.0.0.?

Reply:
every package follows monorepo version from root `package.json`, as it seems as a convention I followed it.
https://github.com/ChainSafe/wallet-selector/blob/b9ca53b3936726497f9962d573b082930a16520f/package.json#L3